analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -81,6 +82,15 @@ class GoMod(
         ) = GoMod(type, analysisRoot, analyzerConfig, repoConfig)
     }
 
+    private val goPath by lazy { createOrtTempDir() }
+
+    private val environment by lazy {
+        mapOf(
+            "GOPROXY" to "direct",
+            "GOPATH" to goPath.absolutePath
```

Comment:
It just occurred to me that there are no changes to expected results in this commit. Do test really still pass as-is with these changes?

Reply:
Yes, they still pass. It'd be quite bad if dependency trees differed depending on whether a proxy was used, wouldn't it?

Reply:
> Yes. But it's Go after all, and as you [explained](https://github.com/oss-review-toolkit/ort/pull/6529/commits/9a09a7af8992d90fa57154048a27033e0efdc8ba) it depends on the proxy instance whether VCS information is available or not (which is quite sick, IMO), so I wouldn't have been surprised if _not_ using a proxy yields different results than before.

My explanation was talking about the presence of VCS info in the `.info` files. 
In this commit, that VCS info from those info files is not yet consumed at all.
This commit only ensures that the info is in these files.